### PR TITLE
Shorten README and add "Getting Started" vignette

### PR DESCRIPTION
**Pull request by jamesmbaazam**
This PR:
* Simplifies the README by removing the function demonstrations and adding descriptions for each function and pointers to available vignettes. 
* Adds a "Getting Started" vignette with improved contents from the README.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -63,251 +63,45 @@ To load the package, use
...
 
-_{{ packagename }}_ provides functionalities for estimating the likelihood of observing a given transmission chain, `likelihood()`, and functions for simulating transmission chains: `simulate_tree()`, `simulate_tree_from_pop()`, and `simulate_summary()`. 
+_{{ packagename }}_ provides four main functions:
 
-The objects returned by these functions play nicely with `summary()` and `aggregate()`. Aggregated results also play nicely with `plot()`.
-Each functionality is briefly demonstrated below. 
+* `simulate_tree()`: simulates transmission chains using an initial number of
+cases and information on the offspring distribution. This function returns
+an object with columns that track information on who infected whom, the
+generation of infection, and optionally, the time of infection.
```
Comment (sbfnk) [inline README.Rmd:73]:
```suggestion
generation of infection and, if a serial interval is given, the time of infection.
```

File: README.Rmd [hunk 2]
```diff
@@ -63,251 +63,45 @@ To load the package, use
...
+lengths using an initial number of cases and information on the offspring
+distribution. This function only returns a vector of realized chain size or
+length.
 
-### [`likelihood()`](https://epiverse-trace.github.io/epichains/reference/likelihood.html)
+* `simulate_tree_from_pop()`: simulates transmission chains given an initial
+population size and information on the offspring distribution. You can also
+specify a given level of pre-existing immunity. This function returns
+an object with columns that track information on who infected whom, the
+generation of infection, and the time of infection.
```
Comment (sbfnk) [inline README.Rmd:84]:
```suggestion
generation of infection and, if a serial interval is given, the time of infection.
```

File: vignettes/epichains.Rmd [hunk 3]
```diff
@@ -27,4 +27,392 @@ knitr::opts_chunk$set(
...
+)
+# Print the estimate
+likelihood_eg
+```
+
+### Joint and individual log-likelihoods
+
+`likelihood()`, by default, returns the joint log-likelihood. If instead,
+the individual log-likelihoods are required, then the `individual` argument
+must be set to `TRUE`. To return likelihoods instead, set `log = TRUE`.
```
Comment (sbfnk) [inline vignettes/epichains.Rmd:85]: should this be `log = FALSE`?

File: vignettes/epichains.Rmd [hunk 4]
```diff
@@ -27,4 +27,392 @@ knitr::opts_chunk$set(
...
+)
+# Print the estimate
+likelihood_ind_eg
+```
+
+### How `likelihood()` works
+
+*epichains* ships with functions for the analytical solutions of some
+transmission chain "size" and "length" distributions. For the size
+distributions, we provide the `poisson`, `negative binomial`, and `gamma-borel`
```
Comment (sbfnk) [inline vignettes/epichains.Rmd:112]: if given in backtics should this not be `pois`, `nbinom` and `gborel`? Whereas if we write them out we should perhaps keep them in normal font (same below)?

**Review by sbfnk — APPROVED** [review]
Looks good - just a few comments.

File: vignettes/epichains.Rmd (hunk 3 above)
Comment (jamesmbaazam) [reply vignettes/epichains.Rmd:85]: Yep, thanks.

File: vignettes/epichains.Rmd (hunk 4 above)
Comment (jamesmbaazam) [reply vignettes/epichains.Rmd:112]: I'll remove the backticks.

**Pull request merged**